duckduckgo/duckduckgo-privacy-extension
# UX for listing unblocked first party requests + entity list changes

**Reviewer:** @andrey-p 




## Description:
MVP UX implementation of the first party unblocked requests list and updates to the entities list.
The unblocked domains paragraph lives in the tracker networks subview, at the bottom. 


## Steps to test this PR:
1. Build and reload extension
2. Go to to a major tracker network's website
3. Open popup
4. Click on the tracker networks line
5. The subview should contain a list of unblocked first party requests at the bottom, separated by a grey top border
6. Also check the unknown company trackers, if applicable: these should be positioned at the bottom, just before the unblocked list 
7. Visit sites like theguardian.com which have their own trackers and are normally listed under unknown - in this case the same domain unblocked trackers should show up in a separate unblocked section
8. Try whitelisting the sites. In that case we still want to show the same domain list, but without "(not blocked)" in the text
9. Run trackers unit tests. These are checking that isTracker returns the correct parent and reason data.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
